### PR TITLE
Fix Spanish copy in a few places

### PR DESCRIPTION
**Pull request by pecanoro**
### Details
Just some copy

### Fixed Issues
$ https://github.com/Expensify/App/issues/17184

### Tests / QA Steps

1. Open the app
2. Enter an email address of an account that already exists.
3. Continue to the next screen
4. Under the `Sign in` button, the copy to go back should say: `Not <email?>? Go back`
Monosnap Emulator - NewExpensify 2023-04-11 20-24-41

5. Change the language to Spanish and repeat the process. Now the copy under `Conectarse` should say: `¿No eres <email>? Volver.`
Monosnap Emulator - NewExpensify 2023-04-11 20-35-05

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

Monosnap New Expensify 2023-04-11 17-16-00
Monosnap New Expensify 2023-04-11 17-16-25

</details>

<details>
<summary>Mobile Web - Chrome</summary>

Monosnap Emulator - NewExpensify 2023-04-11 20-25-26
/231316671-5843d947-2af9-4cd1-bb01-c6302dcae93f.png">

</details>

<details>
<summary>Mobile Web - Safari</summary>

Monosnap iPhone 14 2023-04-11 20-56-22
Monosnap iPhone 14 2023-04-11 20-56-40

</details>

<details>
<summary>Desktop</summary>

Monosnap New Expensify 2023-04-11 17-16-00
Monosnap New Expensify 2023-04-11 17-16-25

</details>

<details>
<summary>iOS</summary>

Monosnap iPhone 14 2023-04-11 20-56-22
Monosnap iPhone 14 2023-04-11 20-56-40

</details>

<details>
<summary>Android</summary>

Monosnap Emulator - NewExpensify 2023-04-11 20-25-26
/231316671-5843d947-2af9-4cd1-bb01-c6302dcae93f.png">

</details>

### CODE REVIEW

**Comment by pecanoro** [comment]
It's all Spanish copy changes, so just leaving @iwiznia as reviewer.

**Review by iwiznia — CHANGES_REQUESTED** [review]

File: src/languages/es.js [hunk 1]
```diff
@@ -606,32 +606,33 @@ export default {
...
     },
     loginForm: {
         phoneOrEmail: 'Número de teléfono o email',
         error: {
             invalidFormatEmailLogin: 'El email introducido no es válido. Corrígelo e inténtalo de nuevo.',
         },
-        cannotGetAccountDetails: 'No se pudieron cargar los detalles de tu cuenta, por favor intenta iniciar sesión de nuevo.',
+        cannotGetAccountDetails: 'No se pudieron cargar los detalles de tu cuenta. Intenta iniciar sesión de nuevo, por favor.',
         loginForm: 'Formulario de inicio de sesión',
+        not: '¿No eres',
```
[the remark this answers is not in the document]
Comment (pecanoro) [reply src/languages/es.js:635]: Sure, I can do that

File: src/languages/es.js [hunk 2]
```diff
@@ -88,15 +88,15 @@ export default {
...
         yesterdayAt: 'Ayer a las',
         conjunctionAt: 'a',
-        genericErrorMessage: 'Ups... algo no ha ido bien y la acción no se pudo completar. Por favor inténtalo más tarde.',
+        genericErrorMessage: 'Ups... algo no ha ido bien y la acción no se ha podido completar. Por favor, inténtalo más tarde.',
         error: {
-            invalidAmount: 'Monto no válido',
+            invalidAmount: 'Importe no válido',
             acceptTerms: 'Debes aceptar los Términos de servicio para continuar',
-            phoneNumber: `Ingresa un teléfono válido, incluyendo el código de país (p. ej. ${CONST.EXAMPLE_PHONE_NUMBER})`,
+            phoneNumber: `Introduce un teléfono válido, incluyendo el código del país (p. ej. ${CONST.EXAMPLE_PHONE_NUMBER})`,
```
[the remark this answers is not in the document]
Comment (pecanoro) [reply src/languages/es.js:95]: Yeah, I used https://www.deepl.com/en/translator to check which would be the most common translation to see what would make more sense.

**Comment by pecanoro** [comment]
Updated, I cleaned even more copy and retested it for the change in the login page.

File: src/languages/es.js [hunk 3]
```diff
@@ -492,16 +492,16 @@ export default {
...
+            invalidName: 'Introduce un nombre válido',
+            addressZipCode: 'Introduce un código postal válido',
+            debitCardNumber: 'Introduce un número de tarjeta de débito válido',
+            expirationDate: 'Introduce una fecha de expiración válida',
+            securityCode: 'Introduce un código de seguridad válido',
+            addressStreet: 'Introduce una dirección de facturación válida que no sea un apartado postal',
+            addressState: 'Seleccione un estado',
+            addressCity: 'Introduce una ciudad',
+            genericFailureMessage: 'Se produjo un error al agregar tu tarjeta. Vuelve a intentarlo',
+            password: 'Introduce tu contraseña de Expensify',
```
Comment (iwiznia) [inline src/languages/es.js:504]: Why less cordial in spanish, not saying please?

File: src/languages/es.js [hunk 4]
```diff
@@ -708,30 +708,30 @@ export default {
...
-            ssnLast4: 'Ingresa los últimos 4 dígitos del número de seguro social',
-            firstName: 'Ingresa el nombre',
-            lastName: 'Ingresa los apellidos',
-            noDefaultDepositAccountOrDebitCardAvailable: 'Por favor agrega una cuenta bancaria para depósitos o una tarjeta de débito',
-            validationAmounts: 'Los montos de validación que ingresaste son incorrectos. Verifica tu cuenta de banco e intenta de nuevo.',
+            ssnLast4: 'Introduce los últimos 4 dígitos del número de seguro social',
+            firstName: 'Introduce el nombre',
+            lastName: 'Introduce los apellidos',
+            noDefaultDepositAccountOrDebitCardAvailable: 'Por favor introduce una cuenta bancaria para depósitos o una tarjeta de débito',
+            validationAmounts: 'Los importes de validación que introduciste son incorrectos. Comprueba tu cuenta bancaria e inténtalo de nuevo.',
```
Comment (iwiznia) [inline src/languages/es.js:734]: Same here, not sure why english says please and spanish does not

File: src/languages/es.js [hunk 5]
```diff
@@ -775,19 +775,19 @@ export default {
     },
     additionalDetailsStep: {
         headerTitle: 'Detalles adicionales',
-        helpText: 'Necesitamos confirmar la siguiente información antes de que podamos procesar este pago.',
-        helpTextIdologyQuestions: 'Tenemos que preguntarte unas preguntas mas para terminar de verificar tu identidad',
+        helpText: 'Necesitamos confirmar la siguiente información antes de que podamos procesar el pago.',
```
Comment (iwiznia) [inline src/languages/es.js:778]: English says `this`, I think this is shown in the flow of attempting to pay a request?

File: src/languages/es.js (hunk 3 above)
Comment (pecanoro) [reply src/languages/es.js:504]: It sounds so weird to say please that many times in Spanish just for error messages.

File: src/languages/es.js (hunk 5 above)
Comment (pecanoro) [reply src/languages/es.js:778]: It's inferred that is referring to the payment that they are trying to perform but I can say `este` again. Up to you

File: src/languages/es.js (hunk 3 above)
Comment (pecanoro) [reply src/languages/es.js:504]: Like it doesn't sound rude or anything at all if we skip the please

Comment (iwiznia) [reply src/languages/es.js:504]: I don't know, same thing could be said about the english phrases and we still have the `please` there 🤷

Comment (pecanoro) [reply src/languages/es.js:504]: Do you want to add Por favor to every error or not? I prefer without it 🤷‍♀️

Comment (iwiznia) [reply src/languages/es.js:504]: I think we either add it to spanish or remove it from english, no? The tone of the copy should be the same in both languages

Comment (pecanoro) [reply src/languages/es.js:504]: I will add it to all of these then 😄

**Comment by pecanoro** [comment]
@iwiznia Ok, went through everything we changed to add more "por favor" when the english copy has it.

**Review by iwiznia — APPROVED** [review]
Thanks!!

**Comment by iwiznia** [comment]
Not emergency, but just copy changes, so I did not do the reviewer checklist

**Pull request merged**